scripts/liquidation_bot/deploy.ts

```diff
@@ -71,6 +75,13 @@ async function main() {
         Exchange.Uniswap,   // WETH
         Exchange.Uniswap,   // UNI
         Exchange.Uniswap    // LINK
+      ],
+      [
+        BigNumber.from(exp(500,18)),    // COMP
```

Comment:
These are the actual values that we'll deploy with.

If we want to be conservative, we could even cut these in half.

Reply:
LINK seems a bit high, we might want to lower it a bit to be safe.

Reply:
Cranked them all down (except for COMP, which was already relatively low). Let me know what you think

Reply:
I think all the other numbers looked fine, so I actually prefer for them to stay the same (other than LINK). IMO, we don't want to be overly cautious with these numbers because it severely hurts the economics of liquidations, both in terms of gas costs and the fact that it takes much longer to liquidate (worst price for both protocol and the liquidator).